src/Controller/AssetsController.php

```diff
@@ -152,23 +154,25 @@ private function getTypePath(string $type, string $slug): string
         return 'images/'.$entity->getMedia()->getPath();
     }
 
-    private function getTypeRepository(string $type)
+    private function getTypeRepository(string $type): ObjectRepository
     {
-        $manager = $this->getDoctrine()->getManager();
-
-        if ('proposal' === $type) {
-            return $manager->getRepository(Proposal::class);
-        }
-
-        if ('clarification' === $type) {
-            return $manager->getRepository(Clarification::class);
-        }
-
-        if ('article' === $type) {
-            return $manager->getRepository(Article::class);
+        switch ($type) {
```

Comment:
Tableau associatif plutôt ?

Reply:
no need to modify this controller in this PR after all !

but yes an array would be a better than the actual code, imho